### PR TITLE
php-cs-fixer native_function_invocation rule

### PR DESCRIPTION
**Pull request by massimilianobraglia**
Introduced php-cs-fixer native_function_invocation rule and applied linting

### CODE REVIEW

**Comment by p365labs** [comment]
@massimilianobraglia :
- why introducing native_function_invocation right now?
- is this process to be applied during linting? if so why no applying during a new release?

**Comment by ruflin** [comment]
This change would also need a CHANGELOG entry. Are there cases where this could break things (certain PHP versions?).

**Comment by massimilianobraglia** [comment]
@p365labs:
1) because we said in https://github.com/ruflin/Elastica/pull/1540 to open multiple PRs to introduce new CS rules
2) yes, it is. Unfortunately I had no time to work on this at the moment the release was created

@ruflin ok about adding a changelog entry (I'll do it right now). The case this could break things is where someone in his codebase mocked the built-in functions I think

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ All notable changes to this project will be documented in this file based on the
 ### Added
 
 ### Improvements
+* Added `native_function_invocation` CS rule
```
Comment (ruflin) [inline CHANGELOG.md]: Can you add the PR number / link to it?

Comment (massimilianobraglia) [reply CHANGELOG.md]: Yep! Sorry about that 😄

**Comment by massimilianobraglia** [comment]
> For the mocking of the built in functions I assume that would be a testing use case?

Yep, exactly

**Comment by p365labs** [comment]
@massimilianobraglia & @ruflin make sense to me. 
- So let's use this on linting
- don't backport it to 6.x branch

**Pull request merged**